vscode-dotnet-runtime-library/src/Utils/CommandExecutor.ts

```diff
@@ -125,13 +125,11 @@ Please install the .NET SDK manually by following https://learn.microsoft.com/en
         this.context?.eventStream.post(new CommandExecutionUserAskDialogueEvent(`Prompting user for command ${fullCommandString} under sudo.`));
 
         // The '.' character is not allowed for sudo-prompt so we use 'NET'
```

Comment:
Move this comment around line 557

Reply:
Good suggestion, I will do that here: https://github.com/dotnet/vscode-dotnet-runtime/pull/1897